### DISCUSSION TITLE
Welcome to mern-social-media Discussions!

### DISCUSSION THREAD

**Discussion by raj074** [raj074/mern-social-media#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rj-robinjoseph** [comment]
![Screenshot 2021-06-12 175041](https://user-images.githubusercontent.com/48469274/121775684-bebd0380-cba6-11eb-9c08-fbcffc0db4ec.png)

Getting errors when I try to run backend server.

**Reply by raj074** [reply]
Check if URI is defined 
And provide mongo database link in .env file.

**Comment by rj-robinjoseph** [comment]


**Reply by raj074** [reply]
paste necessary links at "ADD_LINK_HERE'

**Reply by raj074** [reply]
Now set upload preset and cloud name

**Reply by rj-robinjoseph** [reply]
I didn't saw the links in cloudinary..

**Reply by raj074** [reply]
You will get it from cloudinary dashboard

**Reply by rj-robinjoseph** [reply]
can u please sent a snapshot which shows the upload preset?

**Reply by raj074** [reply]
you will get cloud name from cloudinary dashboard.

**Reply by raj074** [reply]
and for upload preset go to setting -> upload-> Upload presets

**Reply by raj074** [reply]
![Capture](https://user-images.githubusercontent.com/72184791/121776765-4fe2a900-cbac-11eb-81cb-f05cd1a6d415.JPG)

**Reply by rj-robinjoseph** [reply]
![image](https://user-images.githubusercontent.com/48469274/121776809-84eefb80-cbac-11eb-8a27-ed4d600b8896.png)

**Reply by raj074** [reply]
You can use default one or crate another preset

**Reply by raj074** [reply]
Create*

**Reply by rj-robinjoseph** [reply]
What I have to paste, Name?

**Reply by rj-robinjoseph** [reply]
https://discord.gg/uyPMarZn

can you please join the discord server, I will be very grateful.

**Reply by rj-robinjoseph** [reply]
How can I access the admin panel?

**Reply by raj074** [reply]
> What I have to paste, Name?
Yes
In this case ml_default

**Reply by raj074** [reply]
> How can I access the admin panel?

Register user then change its role to "admin" from "user"
Then login as admin from that account  and you will have access to all admin functionalities

**Reply by rj-robinjoseph** [reply]
> https://discord.gg/uyPMarZn
> 
> can you please join the discord server, I will be very grateful.

..

**Reply by raj074** [reply]
[](https://discord.gg/EVDNhGNK) 
Join this server msg me in temp channel

**Reply by raj074** [reply]
https://discord.gg/EVDNhGNK

**Comment by khorshed1032** [comment]
Hello, How to Change Role from USER to Admin?  is it from MongoDB Compass? can't accessing admin panel. help please

**Reply by raj074** [reply]
Yes change it from mongodb compass
Then login as admin.

**Comment by ghost** [comment]


**Reply by raj074** [reply]
It is working perfectly fine. 
Post will appear in dashboard only after more than 1 report. 
So try to report post from 2 accounts.

**Comment by l1msn** [comment]
Hey, raj
Could you tell me how can I log in as Admin?
Do I have to perform manipulations in the mongodb compass?

**Reply by rj-robinjoseph** [reply]
Yes, you have to change it in Mongodb compass. After that, you will be able to login as Admin.

**Comment by yaswith123** [comment]
i'm unable to post the posts

getting an error : 
![image](https://user-images.githubusercontent.com/61270165/177032424-aae9ddec-7b47-4899-8c5b-a11aae01ed33.png)

**Reply by deviil-iish** [reply]
getting the same error, did you find its solution

**Reply by Shabir-Ahmad-Sediqi** [reply]
I got the same error, did you fix it

**Comment by shreyash2509** [comment]
can we connect sometime in discord or anyother platform 
there are some issues for which i need some guidance

**Comment by deviil-iish** [comment]
unable to post the post @raj074 pls help
